src/mpyl/cli/build.py

```diff
@@ -115,15 +117,23 @@ def build(ctx, config, properties, verbose):
     is_flag=True,
     help="Build all projects, regardless of changes on branch",
 )
+@click.option(
+    "--dryrun",
+    "dryrun_",
+    is_flag=True,
+    default=False,
+    help="don't push or deploy images",
```

Comment:
This illustrates why `dryrun` means different things in different contexts. 
I think we should stop passing it along to individual steps, but rather map the CLI flag to what it means locally e.g. don't push image, write manifest instead of deploy, etc.

Reply:
@SamTheisens Yes I see what you mean.. Perhaps better to change in the next PR?